src/sentry/static/sentry/app/components/avatarChooser.tsx

```diff
@@ -157,18 +155,18 @@ class AvatarChooser extends React.Component {
                 choices={choices}
                 value={avatarType}
                 label="Avatar Type"
-                onChange={id => this.handleChange(id)}
+                onChange={this.handleChange}
                 disabled={disabled}
               />
 
               {isLetter && (
                 <Avatar
                   gravatar={false}
                   style={{width: 90, height: 90}}
-                  user={isUser ? model : null}
-                  organization={isOrganization ? model : null}
-                  project={isProject ? model : null}
-                  team={isTeam ? model : null}
+                  user={isUser ? (model as AvatarUser) : undefined}
+                  organization={isOrganization ? (model as Organization) : undefined}
+                  project={isProject ? ((model as unknown) as Project) : undefined}
```

Comment:
Why do you have to double cast here?

Reply:
I considered leaving a comment here. It’s because project does not overlap with Avatar at all (no avatar attribute), since projects do not currently have icons. So I have to cast as unknown first

It wouldn’t surprise me if we don’t use this at all for projects

Reply:
I don't think we do as there is no avatar picker in project settings.

Reply:
Yeah we don't use the avatar picker for projects, we use the platform list

Reply:
Should I drop this prop?

Reply:
okay dropping the support for projects since we don't use it for that.